db/seeds/trust_levels.yml

```diff
@@ -0,0 +1,53 @@
+- name: Participate Generally
+  internal_id: unrestricted
+  icon: fas fa-heartbeat

```

Comment:
Can we make this `heartbeat` and insert the `fas fa-` in views automatically? That way the user/admin doesn't need to know about the internals of how icons work.